MUDscape/driver.py

```diff
@@ -73,6 +82,22 @@ def send(self, message='', prompt=True):
 
         self.request.sendall(message.format(**Fore.__dict__).encode('utf-8'))
 
+    def bufferInput(self):
+        data = bytes()
+        newLine = "b\'\\r\\n\'"
+        backspace = "b\'\\x08\'"
+        delete = "b\'\\x7f\'"
+        #print(newLine, backspace, delete)
+
+        while True:
+            nextChar = self.request.recv(1024)
```

Comment:
Eh, yeah. I just don't want to read in data character by character if a client is sending by line.

Honestly, allocating 1KiB for a buffer isn't really an issue, though. And we still need to do some char-by-char processing.